README.md

```diff
@@ -115,7 +119,7 @@ GitHub: [@githubhandle](https://www.github.com/FrankMugagga)
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[Views]**
+- [ ] **[add devise]**
 
 
```

Comment:
- [ ] It would be nice if you fix the N+1 problems using includes method 👍 Kindly update your controllers to make your app fix the N+1 problems 😊 I can see you have provided the before and after screenshots in the pull request summary but what changes you have done to make your app fix the N+1 problems 😕 Kindly fix this issue 😊